### ISSUE TITLE
"Preview Pre..."

### ISSUE THREAD

**Issue by Transisto** [sparrowwallet/sparrow#434]
![image](https://user-images.githubusercontent.com/388716/154902522-ff2c625c-1650-41a4-a72e-f22b9e4a0caa.png)

That button name might confuse some people.

**Comment by Transisto** [comment]
![image](https://user-images.githubusercontent.com/388716/155609153-6c677138-5689-47c6-a17b-f84c1e95b313.png)

missing a single letter to Passwor...

**Comment by craigraw** [comment]
Is this all occurring on the M1 system? If so will likely need to wait for a native release on the platform to fix.

**Comment by Transisto** [comment]
Yes, M1 but I suspect is could be OSX resolution / scaling specific.

**Comment by craigraw** [comment]
Please retest with the M1 native release: https://github.com/sparrowwallet/sparrow/releases/tag/1.6.4-beta1
